ssh-sk-client.c

```diff
@@ -389,7 +389,9 @@ sshsk_free_resident_keys(struct sshsk_resident_key **srks, size_t nsrks)
 
 	for (i = 0; i < nsrks; i++)
 		sshsk_free_resident_key(srks[i]);
+	if (srks != NULL){
```

Comment:
free(NULL) is perfectly fine according to the C standard (see section 4.10.3.2 of ANSI C).